io.sourceforge.chart_geany.chart-geany.yml

```diff
@@ -0,0 +1,57 @@
+app-id: io.sourceforge.chart_geany.chart-geany
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp # otherwise webengine is not available
+base-version: '5.15-22.08'
+command: ChartGeany
+desktop-file-name-suffix: '' # used to create development version
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+
+# Rename to match the app-id konvention
+rename-icon: chartgeany
+rename-desktop-file: chartgeany.desktop
+rename-appdata-file: chartgeany.metainfo.xml
+
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+
```

Comment:
problably need a `cleanup` rule for includes and other build only files.

Reply:
there are any include files anymore in the application (build folder)